### ISSUE TITLE
Is it possible to trade on margin on Binance

### ISSUE THREAD

**Issue by Denebulas** [knowm/XChange#3557]
I did not find a method/parameter to trade on margin even in the raw version(BinanceTradeServiceRaw)
It is an important feature. is it totally unsupported ?

**Comment by mdvx** [comment]
yes, look at https://github.com/Binance-docs/Binance_Futures_Java

**Comment by badgerwithagun** [comment]
Support could be added in someone raised PRs for it, but it's not currently there.
